app/views/pages/_editor_frontmatter_help.html.erb

```diff
@@ -0,0 +1,29 @@
+<div id="editor-frontmatter-help">
+  <div class="crayons-article">
+    <div class="crayons-article__body">
+      <div class="crayons-notice crayons-notice--info">
+        <p><strong>We have two editor versions</strong>. If you'd prefer to edit title and tags etc. as separate fields, switch to the "rich + markdown" option in <a href="/settings/ux">/settings/ux</a>. Otherwise, continue...</p>
```

Comment:
let's not harcode the path here, you can use `<%= user_settings_path(tab: :ux) %>`. I would also put "Settings -> UX" as the label

Reply:
good to know! thanks and fixed!